plugins/base/src/main/resources/dokka/styles/style.css

```diff
@@ -1181,3 +1181,13 @@ div.runnablesample {
         display: none;
     }
 }
+.clearfix::after {
+    content: ' ';
+    clear: both;
+    display: block;
+    height: 0;
+}
+
+.right-aligned {
+    float: right;
+}
```

Comment:
It's a bit strange that `right-aligned` class sets `float`. I'd expect `TextStyles.RightAligned` to add `align-text: right`

Any particular reason why it shouldn't be called `FloatingRight`?